### PR TITLE
Enrich the targets for common tests and proper checking

### PR DESCRIPTION
**Pull request by HJianBo**
1. Support `make emqx-ct`
2. Support proper checking targets for each application. E.g: `make apps/emqx_web_hook-prop` or `make emqx-prop`

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/emqx/emqx/blob/master/CONTRIBUTING.md.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] In case of non-backward compatible changes, reviewer should check this item as a write-off, and add details in **Backward Compatibility** section

## More information

### CODE REVIEW

**Comment by HJianBo** [comment]
The current use case under `*/test/props` has some failures. I will try to fix them. Once that's done, we can add proper checks to the Github action.

File: scripts/find-props.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+# ensure dir
+cd -P -- "$(dirname -- "$0")/.."
+
+BASEDIR="."
+if [ "$1" != "emqx" ]; then
+    BASEDIR="$1"
+fi
+
+find "${BASEDIR}/test/props" -name "prop_*.erl" -exec basename {} .erl \; 2>/dev/null | xargs | tr ' ' ','
```
Comment (zmstone) [inline scripts/find-props.sh:16]:
```suggestion
find "${BASEDIR}/test/props" -name "prop_*.erl" | xargs basename {} .erl \; 2>/dev/null | xargs | tr ' ' ','
```

Comment (zmstone) [reply scripts/find-props.sh:16]: and `| tr -d '\r' | tr '\n' ','` for `xargs | tr ' ' ','`
or change `find-suites.sh` too for consitency

Comment (HJianBo) [reply scripts/find-props.sh:16]: I prefer the latter. Because the former would append an extra comma at the end.

Comment (zmstone) [reply scripts/find-props.sh]: ok

**Review by zmstone — APPROVED** [review]

**Review by Rory-Z — APPROVED** [review]

**Pull request merged**